provider/offline_test.go

```diff
@@ -213,16 +245,31 @@ func destroyRedshiftDatabase(c RedshiftConfig) error {
 	url := fmt.Sprintf("sslmode=require user=%v password=%s host=%v port=%v dbname=%v", c.Username, c.Password, c.Endpoint, c.Port, "dev")
 	db, err := sql.Open("postgres", url)
 	if err != nil {
+		fmt.Errorf(err.Error())
 		return err
 	}
 	disconnectQuery := fmt.Sprintf("SELECT pg_terminate_backend(pg_stat_activity.procpid) FROM pg_stat_activity WHERE datid=(SELECT oid from pg_database where datname = '%s');", c.Database)
 	if _, err := db.Exec(disconnectQuery); err != nil {
+		fmt.Errorf(err.Error())
 		return err
 	}
+	var deleteErr error
+	retries := 5
 	databaseQuery := fmt.Sprintf("DROP DATABASE %s", sanitize(c.Database))
-	if _, err := db.Exec(databaseQuery); err != nil {
-		return err
+	for {
+		if _, err := db.Exec(databaseQuery); err != nil {
```

Comment:
Its mainly checking checking for if the connections are closed, but regardless its good to let it check anyways incase a weird error happens. Otherwise need to string match a bunch of errors